Automattic/jetpack
# Masterbar Site Menu - Tell Calypso the current site via `?origin_site_id=`

Related to 

- https://github.com/Automattic/wp-calypso/pull/92777
- https://github.com/Automattic/dotcom-forge/issues/8234

## Proposed changes:


This PR adds the `origin_site_id` URL param to the links in the masterbar. 

<img width="1207" alt="Screenshot 2024-07-18 at 15 37 36" src="https://github.com/user-attachments/assets/af4f10fd-500f-4d51-95a8-e0109fde8b42">

This is complemented by https://github.com/Automattic/wp-calypso/pull/92777: It initializes the selected site by reading `origin_site_id`.


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to /wp-admin
* Observe the links are updated
* Ensure the added `origin_site_id` works by using Calypso Live at https://github.com/Automattic/wp-calypso/pull/92777
